CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   FIPS Compliant mode is slightly slower then non-FIPS compliant
   ([cyberark/conjur#1527](https://github.com/cyberark/conjur/issues/1527))
 - Bump conjur-rack from 4.0.0 to 4.2.0 that consumes FIPS compliant slosilo(https://github.com/cyberark/conjur/issues/1527))
+- Print login and authentication error to the log in INFO level 
+  ([cyberark/conjur#1377](https://github.com/cyberark/conjur/issues/1377))
```

Comment:
The guidance for changelog entries [was recently updated](https://github.com/cyberark/community/pull/71). So these should now end with punctuation and not put the link in parens. For example:

```suggestion
- Print login and authentication error to the log in INFO level. 
  [cyberark/conjur#1377](https://github.com/cyberark/conjur/issues/1377)
```

Reply:
thanks, didn't know that.